api/client/commands.go

```diff
@@ -393,6 +393,10 @@ func (cli *DockerCli) CmdLogin(args ...string) error {
 		return err
 	}
 	registry.SaveConfig(cli.configFile)
+
+	// encourage user to use `docker logout` in order to cleanup persisted login credentials
```

Comment:
The comment no longer matches the string that's being output. I don't thing the comment adds much (in its current form), so it might be better to remove it altogether.